addon/components/lio-popover.js

```diff
@@ -197,55 +201,66 @@ export default Component.extend(ParentComponentMixin, ChildComponentMixin, Activ
   reposition: observer('position', 'active', function() {
     if (get(this, 'active')) {
       var $el = this.$();
-      var $arrow = $el.find('.arrow');
-      var $anchor = $(get(this, 'anchor'));
-      var trueAnchorOffset = $anchor.offset();
-      var anchorOffset = get(this, 'alignToParent') ? $anchor.position() : trueAnchorOffset;
-      trueAnchorOffset || (trueAnchorOffset = { top: 0, left: 0});
-      anchorOffset || (anchorOffset = { top: 0, left: 0 });
+      var oldHeight = $el.outerHeight();
+      var oldWidth = $el.outerWidth();
+      this.calculateReposition($el);
+
+      // In cases where positioning alters the element's height we need to run calculations twice.
+      if (oldHeight !== $el.outerHeight || oldWidth !== $el.outerWidth()) {
```

Comment:
I believe that outerWidth should be called. This is just testing for the property